### PR TITLE
feat(docs): add community docs

### LINKED ISSUE #234: Add community docs
Originally created on Thu, 10 Jan 2019 11:36:52 +0200

Contributing guide and later Issue and PR template.

### PR DESCRIPTION
**Pull request by k-dovgan**
Resolves #234

### CODE REVIEW

**Comment by k-dovgan** [comment]
Code of conduct created from Github built-in "Contributor Covenant (Recommended for projects of all sizes)" template.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+or your own [**pull request**](PULL_REQUEST_TEMPLATE.md).
+Using these templates will greatly simplify your interaction with Universum team.
+But **this is not mandatory**. We always welcome any help.
+
+
+## How to make a clean pull request
+
+### 0. [Fork](http://help.github.com/fork-a-repo/) the project repository from Github
+
+This step is requiered because of current Universum access policies: you won't be able to commit
```
Comment (i-keliukh) [inline .github/CONTRIBUTING.md]: "requiered"

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# Contributing
+
+If you want to contribute to a project and make it better, your help is very welcome.
+Contributing is also a great way to learn more about social coding on Github,
+new technologies and and their ecosystems and how to make constructive,
```
Comment (i-keliukh) [inline .github/CONTRIBUTING.md]: "and and"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A detailed explanation of what exactly happened.
+
+**Environment**
+ - OS: [e.g. Ubuntu 16.04.5]
+ - Python version [e.g. 2.7.6]
+
```
Comment (i-keliukh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Universum version?

Comment (k-dovgan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Presumably latest, isn't it?

**Pull request merged**